packages/react-ui/src/app/mcp/mcp-connect.tsx

```diff
@@ -0,0 +1,559 @@
+import { ReloadIcon } from '@radix-ui/react-icons';
```

Comment:
This place can have it's own folder

mcp-connect with index.ts

and then split the component in each file, we strongly recommend one component per file.